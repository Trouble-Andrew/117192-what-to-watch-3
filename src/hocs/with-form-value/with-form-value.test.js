import React from "react";
import renderer from "react-test-renderer";
import withFormValue from "./with-form-value.jsx";

const MockComponent = () => {
  return <div></div>;
};

const MockComponentWrapped = withFormValue(MockComponent);

const user = {
  id: 1,
  email: `gg@mail.com`,
  name: `gg`,
  avatar: `https://htmlacademy-react-3.appspot.com//wtw/static/avatar/8.jpg`,
};

it(`withFormValue is rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped
      user={user}
    />
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
