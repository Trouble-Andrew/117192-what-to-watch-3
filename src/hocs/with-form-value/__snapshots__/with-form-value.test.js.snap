// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withFormValue is rendered correctly 1`] = `<div />`;
